backend/globaleaks/handlers/admin/staticfiles.py

```diff
@@ -121,7 +121,6 @@ def post(self, filename):
         uploaded_file = self.get_file_upload()
         if uploaded_file is None:
             self.set_status(201)
-            self.finish()
             return
```

Comment:
Similar to the other return above. This return closes execution, but the status code should be a 40x not a 201.

Additionally, if this pattern occurs a few times why not make it a function.